package.json

```
@@ -7,11 +7,13 @@
   ],
   "scripts": {
     "test": "mocha tests/test-*.js",
-    "lint": "standard tests/**.js lib/**.js example/**.js"
+    "lint": "standard tests/**.js lib/**.js example/**.js",
+    "lint-fix": "standard tests/**.js lib/**.js example/**.js --fix"
```

Do not repeat the entire command, you can reference it:

```suggestion
    "lint-fix": "npm run lint -- --fix"
```

---

yes.